### DISCUSSION TITLE
change part of url with key-binding?

### DISCUSSION THREAD

**Discussion by av930** [gdh1995/vimium-c#885] [Q&A] [answered]
I usally visit site1, site2, site3 ... in my office everyday. 
site1 is http://jenkins.aec.com/jenkins01/site-config/main.html ... 
site2 is http://jenkins.aec.com/jenkins02/site-config/main.html ...
site3 is http://jenkins.aec.com/jenkins03/site-config/main.html ...
...
I'd like visit site2,site3 after finishing job on site1, and so on.
(site1 has a lot of urls following from jenkins 01/~~)

I want to change only part of url "jenkins01" with key-binding. is it possible?

**Comment by gdh1995** [comment, marked as answer]
Um, you need to add this into `Auto substitution of various text` on Vimium C Options page:
```
n@^http://jenkins\.aec\.com/jenkins(\d+)@http://jenkins.aec.com/jenkins0$${$1,:}@
```

But, it will generate some wrong URLs if the dest number is large than `9` - because of the pattern `0$${$1,:}`, the next url of `09` will be `010`.

I'll fix it in a next version.

**Reply by av930** [reply]
ok. thanks. I have read the manual. Please add more simple examples. I am a newbie, It is little bit hard to understand. please.

**Reply by gdh1995** [reply]
Um sorry but in fact most commands and parameters in Vimium C are not well-documented - I have no much time to spend in writing articles.
